pages/dev/send-tokens/interchain-tokens.mdx

```diff
@@ -0,0 +1,58 @@
+import { Callout } from 'nextra-theme-docs'
+
+# Interchain Tokens
+
+Axelar has [several ways](overview) to send [supported tokens](/resources/testnet#assets), but what if you want to send another ERC20 and have it be available on multiple chains? Axelar can help with our new Interchain Tokens!
+
+Our Interchain Token tools register an existing ERC20 (we'll help you create a new one one if you don't have one) and make it available on multiple chains.
+
+<Callout type="warning" emoji="⚠️">
+ Interchain Tokens are currently only available on testnets. Stay tuned to be among the first to find out about our mainnet launch.
+</Callout>
+
+## Creating an Interchain Token
+The first step is to visit the [https://testnet.services.axelar.dev/](Interchain Token Portal). Select the correct source network in your wallet, and then enter the address of an existing ERC20 token, or choose to create a new one. 
```

Comment:
technically the interchain portal is a part of the "maestro" series of products that helps you manage contracts across multipe chains. do we want to rename the portal to "Interchain Maestro" then?